### PR TITLE
DOCS-2723: Update introduction page with new features

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/payments/checkout/payment-components/_index.md [hunk 1]
```diff
@@ -10,14 +10,21 @@ weight: 51
...
+- Shift responsibility for [PCI DSS compliance](/faq/general/multisafepay-glossary/#payment-card-industry-data-security-standard-pci-dss) to MultiSafepay
+- Customize the component interface to match your brand's visual identity
 
 {{< screen src="/gifs/credit-card-component.gif" alt="Credit Card Component" align="center" class="medium-img" screen_size="desktop" >}}
 
+## Payment methods
+
+Embed a single payment method or multiple methods, which are automatically sorted by availability in the customer's country. 
+
+The customer completes payment within your checkout or is redirected to the [issuer's](/getting-started/glossary/#issuer) payment page.
```
Comment (Kris-MultiSafepay) [inline content/payments/checkout/payment-components/_index.md:26]: Currently, this still holds, but soon customers aren't redirected only to issuers e.g. PayPal.

File: content/payments/checkout/payment-components/_index.md [hunk 2]
```diff
@@ -10,14 +10,21 @@ weight: 51
...
+- Create a seamless checkout experience to increase [conversion](/getting-started/glossary/#conversion-rate)
+- Encrypt customer payment details for secure processing
+- Shift responsibility for [PCI DSS compliance](/faq/general/multisafepay-glossary/#payment-card-industry-data-security-standard-pci-dss) to MultiSafepay
+- Customize the component interface to match your brand's visual identity
 
 {{< screen src="/gifs/credit-card-component.gif" alt="Credit Card Component" align="center" class="medium-img" screen_size="desktop" >}}
 
+## Payment methods
+
+Embed a single payment method or multiple methods, which are automatically sorted by availability in the customer's country.
```
Comment (Kris-MultiSafepay) [inline content/payments/checkout/payment-components/_index.md:24]: They're filtered by availability and sorted by popularity

**Review by Kris-MultiSafepay — CHANGES_REQUESTED** [review]

File: content/payments/checkout/payment-components/_index.md (hunk 1 above)
Comment (FrancescaNCook) [reply content/payments/checkout/payment-components/_index.md:26]:
```suggestion
The customer completes payment within your checkout or is redirected to the relevant [payment page](/getting-started/glossary/#payment-page). 
```

Comment (FrancescaNCook) [reply content/payments/checkout/payment-components/_index.md:26]: @Kris-MultiSafepay Does this future-proof it better?

File: content/payments/checkout/payment-components/_index.md (hunk 2 above)
Comment (FrancescaNCook) [reply content/payments/checkout/payment-components/_index.md:24]:
```suggestion
Embed a single payment method or multiple methods, which are automatically filtered by availability in the customer's country and sorted by popularity. 
```

**Review by Kris-MultiSafepay — APPROVED** [review]

**Pull request merged**